### PR TITLE
chore(): Delete message in exception constructors

### PR DESCRIPTION
**Pull request by k-sauvee**
#### Issue Number
fixes #

#### Checklist
- [ ] Refer to issue(s) the PR solves
- [ ] New java code is covered by tests
- [ ] Add screenshots or gifs of the new behavior, if applicable.
- [x] All new and existing tests pass
- [x] No git conflict

### CODE REVIEW

File: chutney/engine/src/main/java/com/chutneytesting/engine/domain/execution/engine/evaluation/EvaluationException.java [hunk 1]
```diff
@@ -7,14 +7,20 @@
...
+    }
+
+    EvaluationException(String expression, Exception e) {
+        super("Cannot resolve " + expression + " , " + e.getMessage(), e);
     }
 
-    EvaluationException(String message, Exception cause) {
-        super(message, cause);
+    EvaluationException(String expression, ParseException e) {
+        super("Cannot resolve " + expression + " , " + e.getMessage(), e);
```
Comment (boddissattva) [inline chutney/engine/src/main/java/com/chutneytesting/engine/domain/execution/engine/evaluation/EvaluationException.java:24]: resolve -> parse

File: chutney/environment/src/main/java/com/chutneytesting/environment/domain/EnvironmentService.java [hunk 2]
```diff
@@ -112,7 +112,7 @@ public String defaultEnvironmentName() throws EnvironmentNotFoundException {
             throw new UnresolvedEnvironmentException("There is more than one environment. Could not resolve the default one");
```
Comment (boddissattva) [inline chutney/environment/src/main/java/com/chutneytesting/environment/domain/EnvironmentService.java:112]: Is refactoring here could'nt be done ?

File: chutney/environment/src/test/java/com/chutneytesting/environment/api/environment/HttpEnvironmentApiTest.java [hunk 3]
```diff
@@ -156,7 +156,7 @@ public void deleteEnvironment_does_not_deletes_last_env() throws Exception {
 
     @Test
     public void deleteEnvironment_returns_404_when_not_found() throws Exception {
-        doThrow(new EnvironmentNotFoundException("message")).when(environmentRepository).delete(any());
+        doThrow(new EnvironmentNotFoundException(List.of("message"))).when(environmentRepository).delete(any());
```
Comment (boddissattva) [inline chutney/environment/src/test/java/com/chutneytesting/environment/api/environment/HttpEnvironmentApiTest.java:159]: We could have a one string constructor for this.

**Comment by boddissattva** [comment]
We had a discussion on https://github.com/Enedis-OSS/chutney/blob/695c7659cf28177fd3be25f6aed76ed74d0ed22b/kotlin-dsl/src/main/kotlin/com/chutneytesting/kotlin/junit/engine/execution/ChutneyScenarioExecutionContext.kt#L182.
Maybe we could take it into this refacoring ?

**Review by nbrouand — APPROVED** [review]

**Review by boddissattva — APPROVED** [review]

**Pull request merged**